context/src/test/java/io/opentelemetry/context/internal/shaded/WeakConcurrentMapTest.java

```diff
@@ -152,7 +152,7 @@ void doTest() throws Exception {
       assertThat(values.isEmpty(), is(true));
       key1 = key2 = null; // Make eligible for GC
       System.gc();
-      Thread.sleep(200L);
+      Thread.sleep(1000L);
```

Comment:
could we also change the asserts below to be under an `await()` ?

Reply:
I _think_ this sleep is more about waiting for the gc to complete before running `triggerClean()` below

Reply:
alternatively, I could add the `triggerClean()` into the `await()` block, I'll try that